.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "4"
   - "5"
```

Comment:
I'm happy to drop 5 as well at this point.

Probably worth adding the appropriate alias for latest stable as well